### ISSUE TITLE
calculate of social_work

### ISSUE THREAD

**Issue by hatta-yuuto** [robotics-upo/social_force_window_planner#11]
Hi.
I have a question about the calculation of social_work.
I think the social_work used to calculate the cost of the generated trajectory candidates is calculated from the social_force and obstacle_force  that calculated in sfm.hpp.
Also, as far as I can see in the program, I think social_force is calculated from the distance between the robot and the pedestrian, but at what timing is this distance?
In other words, the trajectory has a certain length, but I couldn't understand whether it is the distance at the end of the trajectory, the beginning of the trajectory, or a certain point in the middle of the trajectory when I read the program.

**Comment by noeperez** [comment]
Hello,

your question is clearly answered in [Line 598](https://github.com/robotics-upo/social_force_window_planner/blob/02fa5c747384331c442d59f908261bb5a673e0e5/src/sfw_planner.cpp#L598) of sfw_planner.cpp.
As you can see, the social work is computed and added for each time step of the trajectory evaluated. 
So, please, try to take a look to the code carefully before asking this kind of questions. 

Beyond that, I must say that this package is intended for people with knowledge of programming, ROS and its navigation stack. Moreover, Github issues is a tool for tracking the development of the code, indicating possible bugs and next steps in the development. So, this is not the most appropriate place for introductory explanations and guides. The required information is included in the readme page and the code itself.

**Comment by hatta-yuuto** [comment]
Thank you.
